tests/unit/test_contrib/test_pydantic/test_beanie_integration.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+import pydantic
+import pytest
+
+from litestar.contrib.pydantic import PydanticDTO
+
+
+@pytest.mark.skipif(pydantic.VERSION.startswith("2"), reason="Beanie does not support pydantic 2 yet")
```

Comment:
Beanie supports pydantic 2 in its latest version.

Reply:
We should test that then!

Reply:
PR welcome